ui/components/molecules/M002-CheckboxWithLabel/stories.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+
+// Import index.js from the same folder
+import FormControlLabel from '.'
+import Checkbox from '../../atoms/A011-Checkbox'
```

Comment:
@RichardGarcia just to draw your attention to this - you dont need to do this. the component your importing from index.js already has a <Checkbox/> inside it.